llvm/tools/llvm-rc/ResourceScriptParser.cpp

```diff
@@ -238,7 +238,27 @@ Expected<StringRef> RCParser::readString() {
 Expected<StringRef> RCParser::readFilename() {
   if (!isNextTokenKind(Kind::String) && !isNextTokenKind(Kind::Identifier))
     return getExpectedError("string");
-  return read().value();
+  const RCToken &Token = read();
+  StringRef Str = Token.value();
+  if (Token.kind() != Kind::String)
+    return Str;
+  while (isNextTokenKind(Kind::String)) {
+    const RCToken &NextToken = read();
+    StringRef Next = NextToken.value();
+    bool IsWide = Str.starts_with_insensitive("L");
+    if (IsWide)
+      Str = Str.drop_front();
+    if (Next.starts_with_insensitive("L"))
+      Next = Next.drop_front();
```

Comment:
This could use `consume_front_insensitive`.

Reply:
Updated to use `consume_front_insensitive`, that does indeed make it yet a little bit nicer. Thanks!